Packs/DB2/Integrations/DB2/DB2.yml

```diff
@@ -0,0 +1,62 @@
+category: Database
+commonfields:
+  id: DB2
+  version: -1
+configuration:
+- display: Database host
+  name: host
+  required: true
+  type: 0
+- display: Port
+  name: port
+  required: false
+  type: 0
+- display: Database Name
+  name: dbname
+  required: false
+  type: 0
+- display: Username
+  name: credentials
+  required: true
+  type: 9
+- display: 'Connection Arguments (ex: arg1=val1&arg2=val2)'
+  name: connect_parameters
+  required: false
+  type: 0
+- display: Use an SSL connection
+  name: ssl_connect
+  required: false
+  type: 8
+- display: Use Persistent Connection
+  name: use_persistent
+  required: false
+  type: 8
+description: Integration to provide connectivity to IBM DB2 using the python ibm_db2 library.
+display: DB2
+name: DB2
+script:
+  commands:
+  - arguments:
+    - description: The DB2 query to run
+      name: query
+      required: true
+    - defaultValue: "50"
+      description: The maximum number of results
+      name: limit
+    - defaultValue: "0"
+      description: The offset at which to start the result
+      name: skip
```

Comment:
```suggestion
      name: offset
```

Reply:
renamed in [3fe28ed](https://github.com/demisto/content/pull/18139/commits/3fe28edaae14abdcd24bc1f718d86478d6bfd3ab)